brakeman_report.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE HTML SYSTEM>
```

Comment:
Can we add this report to `.gitignore` ? Or is there a need to have this inside the repo?